Dockerfiles/compliance-operator-content-konflux.Containerfile

```diff
@@ -60,6 +60,9 @@ RUN if [ "$(uname -m)" = "x86_64" ]; then \
 	sed -i 's/\(documentation_complete: \).*/\1true/' products/ocp4/profiles/stig-v2r1.profile && \
 	sed -i 's/\(documentation_complete: \).*/\1true/' products/ocp4/profiles/stig-node-v2r1.profile && \
 	sed -i 's/\(documentation_complete: \).*/\1true/' products/rhcos4/profiles/stig-v2r1.profile; \
+    elif [ "$(uname -m)" = "ppc64le" ]; then \
+        find products/rhcos4 -name "*stig*.profile" | xargs sed -i 's/\(documentation_complete: \).*/\1true/' && \
```

Comment:
@prb112 I think you need to add `ppc64le` in the `if` condition on line 76, otherwise `RHCOS4` content won't be built.

Reply:
Absolutely correct, fixing now